indico/modules/events/templates/emails/event_creation.txt

```diff
@@ -0,0 +1,42 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    New {{ event.type }} in Indico ({{ event.id }})
+{%- endblock %}
+
+{% block header -%}{%- endblock %}
+
+{% macro render_dates_and_access(event) %}
+    _Date_
+        {{ event.start_dt_local|format_date }} -> {{ event.end_dt_local|format_date }}
+    _Access_
+        {{ url_for('event.conferenceDisplay', event, _external=true) }}
+{% endmacro %}
+
+{% block body -%}
+    {%- filter dedent -%}
+        _Category_
+            {{ event.category.chain_titles|join(' \N{RIGHT-POINTING DOUBLE ANGLE QUOTATION MARK} ') }}
```

Comment:
Just put the actual character here. We only use the named unicode chars in python files to keep them plain ASCII